### ISSUE TITLE
How many epochs for training

### ISSUE THREAD

**Issue by Karthikaddagadderamesh** [fuy34/planerecover#16]
Hello,

This is not a issue but a clarification.

I was able to setup the training for the pre processed images. I ran till 2000 steps that is 1 epoch. I wanted to know how many epochs the model has to be trained for as in the paper it mention the model converged for 500k iteration.
As one epoch is 1000 steps for the model to converge the training needs to run till 500k steps.

The tensor board graph shows after 1 epoch to be 2000k hence I am confused. 
The model needs 500k steps that is around 250 epochs to converge. Is my understanding right?

**Comment by fuy34** [comment]
Hi, 

I am not sure what is going on in the tensorboard part. But you are right, there are 8000 images in the train set, and our batch size is 4. So it should be 2000 steps per epoch, which means 500K/ 2k = 250 epochs.

I wrote the code a few years ago, and honestly, I did not consider the number of epochs at that time. The total training step is controlled by this line of code, where I set ```opt.max_steps=500K``` in ```train.py```.
https://github.com/fuy34/planerecover/blob/3cbba0a6db9d722ece827ea0f3b1dbcb609ca5f9/RecoverPlane.py#L252
